theory/Theory.md

```diff
@@ -0,0 +1,22 @@
+Jason's Ramblings
+--
+
+Is a `ChunkDB` a theory presentation? Maybe we should flush out this discussion
```

Comment:
No, it's not.

Theory presentations contain:
- definitions of domains of discourse
- vocabulary, potentially annotated with various properties 
- "laws", i.e. relations between the meaning of the parts of the vocabulary

I defined Theory Presentation above in a generic enough way that it be able to encompass both purely formal theories and informal theories at the natural language level (and hopefully many things in between).